docs/csharp/language-reference/keywords/using-directive.md

```diff
@@ -46,9 +46,9 @@ class Program
 ```  
   
 ## Remarks  
- The scope of a `using` directive is limited to the file in which it appears.  
+ The scope of a `using` directive is limited to the file in which it appears. It can be declared in any namespace (including the default namespace) but before any namespace member, otherwise compiler error [CS1529](../../misc/cs1529.md) is generated.
```

Comment:
A couple of comments:
- The "It can be declared" should begin a new paragraph.
- The implication is that the directive has to be inside a namespace. I'd change what will now be the second to paragraph to read something like:

The \`using\` directive can appear:
- At the beginning of a source code file, before any namespace or type definitions.
- In any namespace, but before any namespace type definitions.
Otherwise, compiler error...